### PR TITLE
Fix linter in project

### PR DESCRIPTION
**Pull request by guilleSequeiraWolox**
## Summary

a) Modified Prettier.js and esLintrc.
b) adjusted the files according to the new eslint rules

## Trello Card

https://trello.com/c/r3ADx675/121-arreglar-linter-en-el-proyecto-generado

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -2,6 +2,19 @@ module.exports = {
...
+    'init-declarations': 'off',
+    "no-empty-function": "off",
+    'complexity': 'off',
+    'no-extra-parens': 'off',
+    'new-cap': 'off',
+    'no-shadow': 'off',
+    'react/forbid-foreign-prop-types': 'off',
+    'prefer-promise-reject-errors': 'off',
+    'camelcase': 'off',
+    "react/jsx-closing-bracket-location": 'off'
```
Comment (WFolini) [inline .eslintrc.js]: Why all these rules? all these are extended in `wolox-react`

File: generators/app/tasks/installDependencies.js [hunk 2]
```diff
@@ -5,7 +5,6 @@ const runCommand = require('./runCommand');
 
 const DEPENDENCIES = [
   '@react-native-community/async-storage',
-  '@react-native-community/eslint-config',
```
Comment (WFolini) [inline generators/app/tasks/installDependencies.js]: Nice catch

File: generators/app/templates/src/app/components/CustomTextInput/components/ShowPassword/index.js [hunk 3]
```diff
@@ -14,8 +14,8 @@ const ShowPassword = ({ onShowPassword, passwordVisible }) => (
 );
 
 ShowPassword.propTypes = {
-  onShowPassword: PropTypes.func.isRequired,
-  passwordVisible: PropTypes.bool.isRequired
+  passwordVisible: PropTypes.bool.isRequired,
+  onShowPassword: PropTypes.func.isRequired
```
Comment (guidoprinc) [inline generators/app/templates/src/app/components/CustomTextInput/components/ShowPassword/index.js]: mmm i think it was okay before

File: generators/app/templates/src/redux/store.ejs [hunk 4]
```diff
@@ -7,15 +7,14 @@ import {
...
-  <%_ } _%>
+import AsyncStorage from '@react-native-community/async-storage';
+import { persistReducer } from 'redux-persist';
+import {
+  seamlessImmutableReconciler,
+  seamlessImmutableTransformCreator
+} from 'redux-persist-seamless-immutable';
+<%_ } _%>
 
-import Navigator from '@screens';
```
Comment (guidoprinc) [inline generators/app/templates/src/redux/store.ejs]: Why did you move this line?

File: generators/app/templates/src/app/components/CustomTextInput/components/ShowPassword/index.js (hunk 3 above)
Comment (guilleSequeiraWolox) [reply generators/app/templates/src/app/components/CustomTextInput/components/ShowPassword/index.js]: it occurs beacuse the react/sort-prop-types puts callbacksLast in true. I think that it is not bad.
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/sort-prop-types.md

File: generators/app/templates/src/redux/store.ejs (hunk 4 above)
Comment (guilleSequeiraWolox) [reply generators/app/templates/src/redux/store.ejs]: when I apply the esLint of react-native-community, it obligate me to sort imports in this file

File: generators/app/tasks/appSetup/index.js [hunk 5]
```diff
@@ -35,9 +36,14 @@ module.exports = function index() {
   // ----------------     babelrc setup     ----------------
   babelConfigSetup.bind(this)();
 
+
```
Comment (sfernandez11) [inline generators/app/tasks/appSetup/index.js]: Delete space

File: generators/app/tasks/appSetup/index.js [hunk 6]
```diff
@@ -35,9 +36,14 @@ module.exports = function index() {
...
+  // ----------------     prettierrc setup     ----------------
+  prettierrcConfigSetup.bind(this)();
+
   // ----------------     add react-native-config to app/build.gradle     ----------------
   // ----------------     add react-native-screen to app/build.gradle     ----------------
   // ----------------     add react-native-gesture-handler to MainActivity    ----------------
+
```
Comment (sfernandez11) [inline generators/app/tasks/appSetup/index.js]: Delete space
